### DISCUSSION TITLE
The chain of multiple tasks

### DISCUSSION THREAD

**Discussion by pesht** [cline/cline#599] [Feature Requests]
First of all, thank you for this great tool!
I think the following feature would be helpfull if it still hasn't implemented (I could have missed something)
When I run complex task that includes for example making nue feature and testing it, I often hit my token limit. It would be nice to be able to create a set of tasks that depends on each other but each sends the new prompt to the model. This new prompt could be the condensed summary of the previous task completion result.

**Comment by d0lwl0b** [comment]
We can jointly implement a "mandatory" standard for LLMs.

see: #5912
